specification/applicationinsights/resource-manager/readme.md

```diff
@@ -28,7 +28,7 @@ These are the global settings for the ApplicationInsights API.
 title: ApplicationInsightsManagementClient
 description: Composite Swagger for Application Insights Management Client
 openapi-type: arm
-tag: package-2021-10
```

Comment:
Just realize you are downgrade the tag version?

Reply:
update to new tag=package-2021-11-01